apps/prairielearn/src/server.js

```diff
@@ -165,6 +165,24 @@ module.exports.initExpress = function () {
     }),
   );
 
+  app.use((req, res, next) => {
+    const issuedAt = new Date(req.session.cookie.expires - req.session.cookie.originalMaxAge);
+
+    // If the cookie was issued more than an hour in the past, refresh it
+    // with a new expiration date.
+    if (issuedAt < config.sessionStoreExtendCooldownSeconds * 1000) {
```

Comment:
Bikeshed on the `sessionStoreExtendCooldownSeconds` name?

Reply:
`sessionMinimumRefreshIntervalSeconds`?